### PR TITLE
Update staff-recruiting.md

### PR DESCRIPTION
**Pull request by natalie-hub**
Updated with new roles, people and processes

### CODE REVIEW

File: contributors/join-us/staff-recruiting.md [hunk 1]
```diff
@@ -154,8 +148,7 @@ Typically the AA is the last person on the interview loop and scheduled in advan
 
 The following list of people are approved as As Appropriate interviewers:
 
-* Ian Tien, CEO and co-founder for all roles outside of R&D.
-* Corey Hulen, CTO and co-founder for all roles in the R&D department including Engineers, Product Managers, and UX.
+* Ian Tien, CEO and co-founder for all roles outside of R&D
```
Comment (lynn915) [inline contributors/join-us/staff-recruiting.md:151]: Ian Tien, CEO and co-founder.

File: contributors/join-us/staff-recruiting.md [hunk 2]
```diff
@@ -201,11 +194,11 @@ Once reference checks are complete and an offer is ready to be made, the Recruit
 
 The order of approval will follow:
 
-1. HR Director
+1. VP, HR
 2. Team MLT member
-3. Either CTO \(for R&D and Customer Engineer offers\) or CEO \(for all other offers\) 
+3. CEO
```
Comment (lynn915) [inline contributors/join-us/staff-recruiting.md]: 3. CFO
4. CEO

File: contributors/join-us/staff-recruiting.md [hunk 3]
```diff
@@ -217,7 +210,7 @@ This process will be managed by HR.
 
 1. Once an offer is approved, the Recruiter will select the appropriate offer template based on candidates’s residence country/region and send for signatures.
```
Comment (lynn915) [inline contributors/join-us/staff-recruiting.md]: Once an offer is approved, HR Generalist will select....

**Review by lynn915 — DISMISSED** [review]
I added a couple of small additions/edits based on our current process but these can be disregarded if no longer relevant.

File: contributors/join-us/staff-recruiting.md [hunk 4]
```diff
@@ -217,7 +210,7 @@ This process will be managed by HR.
...
 2. Upon full execution of a new staff contract, the Recruiter will mark the candidate as **Hired** in Lever and export the profile to BambooHR.
```
Comment (yomiadetutu1) [inline contributors/join-us/staff-recruiting.md:212]: Upon full execution of a new staff contract, the Recruiter will mark the candidate as **Hired** in Lever.

Comment (yomiadetutu1) [inline contributors/join-us/staff-recruiting.md]:
```suggestion
2. Upon full execution of a new staff contract, the Recruiter will mark the candidate as **Hired** in Lever.
```

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**